packages/documentation/plugins/mermaid-svg-remover/gatsby-node.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+const path = require('path');
+
+exports.onPreBootstrap = async () => {
+  const staticDir = path.join(__dirname, '../../static/mermaid-svgs');
+  const publicDir = path.join(__dirname, '../../public/mermaid-svgs');
+
+  await fs.readdir(staticDir, async (err, files) => {
+    if (err) throw err;
+
+    for (const file of files) {
+      await fs.unlink(path.join(staticDir, file), err => {
```

Comment:
Does this work? I don't think this method returns a promise, does it? I guess I have similar questions about the rest of this code, too.

Reply:
@jbalboni Yeah your right this should return a promise. Wow I'm rusty with Node.